### PR TITLE
user specific notebooks

### PR DESCRIPTION
**Pull request by bgruening**
@sanjaysrikakulam @mira-miracoli do you think this will work?

### CODE REVIEW

**Review by sanjaysrikakulam — APPROVED** [review]
This should work, for [reference](https://github.com/usegalaxy-au/infrastructure/blob/6e661620f5b525676c297db044ef244e7ae8c4aa/files/galaxy/dynamic_job_rules/production/total_perspective_vortex/users.yml.j2#L112-L128).

However, I think the `params`, `env`, and even the `scheduling` tag might not be necessary as those will probably be merged from the IT tool spec of `interactive_tool_jupyter_notebook` in `interactive_tools.yml` and the rest from the `destination.yml` file (I think let's go with this for now, and when I get the time, I will double-check it on my test instance).

File: files/galaxy/tpv/users.yml [hunk 1]
```diff
@@ -6,7 +6,8 @@ users:
   andreas.kalagasidis@gmail.com:
     rules:
       - id: user-specific-notebook
-        if: 'interactive_tool_jupyter_notebook' in tool.id
+        if: |
+          'interactive_tool_jupyter_notebook' in tool.id
         cores: 1
```
Comment (bgruening) [inline files/galaxy/tpv/users.yml:11]: Wow, this is wired, shouldn't it be the same from YAML perspective?
Thanks for fixing!

**Comment by bgruening** [comment]
What do you think about adding this to my user? So I can test it ...?

File: files/galaxy/tpv/users.yml (hunk 1 above)
Comment (sanjaysrikakulam) [reply files/galaxy/tpv/users.yml:17]: My best guess is that it was due to the single-quoted string.

**Comment by sanjaysrikakulam** [comment]
> What do you think about adding this to my user? So I can test it ...?

Sure! Let's do it in production! Why not....

**Pull request merged**